tests/source/macros.rs

```diff
@@ -0,0 +1,9 @@
+#![crate_type = "lib"]
+
+// @has /data/relationships/macros/data/0 'macro'
```

Comment:
I can't get this to pick up at all. It doesn't seem to see macros. I have the same issue with unions and I imagine would have the same issue with statics and consts.

Reply:
@steveklabnik what about this problem?

I'm seeing in #7 a comment"After a quick look at generated JSON, it looks like the exported macros (and their doc comments) are not included.". So that kind of seems like a problem. And I've also seen the same issue with unions.

I'm going to tweak the `lib.rs` file with all these things and see what it actually generates because it doesn't seem like a lot right now.

Reply:
sounds like maybe the same thing as impls, I'm not sure.